package.json

```diff
@@ -32,6 +32,7 @@
     "unit": "bash lighthouse-core/scripts/run-mocha.sh --default",
     "closure": "cd lighthouse-core && node closure/closure-type-checking.js",
     "devtools": "bash lighthouse-core/scripts/roll-to-devtools.sh",
+    "compdevtools": "bash lighthouse-core/scripts/compile-against-devtools.sh",
```

Comment:
I know you love 🖐 but I didn't know what this meant haha, can we do `compile-devtools`? it's only 4 more characters :)